packages/olo-gulp-helpers/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "olo-gulp-helpers",
-  "version": "0.1.1",
+  "version": "0.1.2",
```

Comment:
Only real nit would be that I think we should bump this to `0.2.0` since it bumps `gulp-sass` by a major version.

Reply:
done